### PR TITLE
Readd voice server

### PR DESCRIPTION
**Pull request by pupnewfster**
## Changes proposed in this pull request:

- Add back the voice server (context can be found here: https://github.com/mekanica/mekanica/pull/137 ) disabled by default
- Cleaned the voice server up so that all the variables in it are not public
- Voice connections now keep track of the uuid of a player rather than the username. (Shouldn't matter given they have to reconnect when changing usernames but it is better to be consistent)

### CODE REVIEW

File: src/main/java/mekanism/client/ClientConnectionHandler.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    public void onConnection(ClientConnectedToServerEvent event) {
+        if (general.voiceServerEnabled) {
+            if (event.isLocal()) {
+                // If the client is connecting to its own corresponding integrated server.
+                try {
+                    MekanismClient.voiceClient = new VoiceClient("127.0.0.1");
+                    // Will probably not work when multiple integrateds are running on one computer
+                } catch (Throwable e) {
+                    Mekanism.logger.error("Unable to establish VoiceClient on local connection.");
+                    e.printStackTrace();
```
Comment (thommy101) [inline src/main/java/mekanism/client/ClientConnectionHandler.java]: You can combine these, log the throeable with the error.

Comment (pupnewfster) [reply src/main/java/mekanism/client/ClientConnectionHandler.java]: Thanks seems I missed a couple spots that it could be collapsed in.

**Comment by thiakil** [comment]
Looks fine at a surface level, I'll do a more thorough check later

**Comment by pupnewfster** [comment]
Confirmed with Aidan and the reason the config option is different is so that it defaults to disabled in existing installs.

**Pull request merged**